### PR TITLE
Subnet labels

### PR DESCRIPTION
**Pull request by rsagasthya**
**Reason for Change**:

**Issue Fixed**:

**Requirements**:

- [ ] uses [conventional commit messages](https://www.conventionalcommits.org/)
  
- [ ] includes documentation
- [ ] adds unit tests

**Notes**:

### CODE REVIEW

File: cns/ipampool/monitor.go [hunk 1]
```diff
@@ -146,7 +154,9 @@ func (pm *Monitor) reconcile(ctx context.Context) error {
 	allocatedIPs := pm.httpService.GetPodIPConfigState()
 	state := buildIPPoolState(allocatedIPs, pm.spec)
 	logger.Printf("ipam-pool-monitor state %+v", state)
-	observeIPPoolState(state, pm.metastate)
+
+	labels := []string{subnet, subnetCIDR}
+	observeIPPoolState(state, pm.metastate, labels)
```
Comment (rbtr) [inline cns/ipampool/monitor.go:159]: can you inline this - we don't need a local variable `lables` - and remove the whitespace?

**Review by rbtr — CHANGES_REQUESTED** [review]
one change pls

File: cns/ipampool/monitor.go (hunk 1 above)
Comment (rsagasthya) [reply cns/ipampool/monitor.go]: Made this change. I have pushed the necessary changes.

File: cns/ipampool/metrics.go [hunk 2]
```diff
@@ -5,66 +5,81 @@ import (
 	"sigs.k8s.io/controller-runtime/pkg/metrics"
 )
 
+const (
+	subnetLabel     = "Subnet"
+	subnetCIDRLabel = "SubnetCIDR"
```
Comment (rbtr) [inline cns/ipampool/metrics.go:10]: snake-case [for consistency](https://github.com/paulgmiller/azure-container-networking/blob/17258152314ee33986afbd8c011343d72d4511b1/cns/restserver/metrics.go#L39).

**Review by rbtr — CHANGES_REQUESTED** [review]

**Review by rbtr — APPROVED** [review]

**Pull request merged**